### PR TITLE
OSOE-91: Add gitattributes

### PR DESCRIPTION
**Pull request by sarahelsaig**
[OSOE-91](https://lombiq.atlassian.net/browse/OSOE-91)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+wwwroot/
 node_modules/
-.pnpm-debug.log
-package-lock.json
 Localization/
-/.editorconfig
\ No newline at end of file
+*.user
+.pnpm-debug.log
+/.editorconfig
```
Comment (0liver) [inline .gitignore]: I've restored `Localization/` and `/.editorconfig` here.

**Review by 0liver — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (sarahelsaig) [reply .gitignore]: Are they needed though? I assumed they are outdated, because I did a rebuild and nothing materialized.

Comment (0liver) [reply .gitignore]: Well, I take your word for it. I'll remove them then.

Comment (0liver) [reply .gitignore]: Is the deletion in the other two submodules also intended then?

Comment (sarahelsaig) [reply .gitignore]: Yes.

Comment (0liver) [reply .gitignore]: Reverted my commits.

**Pull request merged**